rules/aws_cloudtrail_rules/aws_ec2_traffic_mirroring.yml

```diff
@@ -0,0 +1,516 @@
+        eventType: AwsApiCall
+        eventVersion: "1.08"
+        managementEvent: true
+        p_any_aws_account_ids:
+            - "123451234512"
+        p_any_aws_arns:
+            - arn:aws:iam::123451234513:role/MakeStuffPublic
+            - arn:aws:sts::123451234513:assumed-role/MakeStuffPublic
+        p_any_domain_names:
+            - AWS Internal
+        p_any_trace_ids:
+            - ASIA57JLR4M2ZZDJUXY3
+        p_any_usernames:
+            - MakeStuffPublic
+        p_event_time: "2022-11-15 22:58:13"
+        p_log_type: AWS.CloudTrail
+        p_parse_time: "2022-11-15 23:00:37.356"
+        p_row_id: 82670d2e7575bbd0e8fc97d014b6a80c
+        p_source_id: 125a8146-e3ea-454b-aed7-9e08e735b670
+        p_source_label: Panther Identity Org CloudTrail
+        readOnly: false
+        recipientAccountId: "123451234566"
+        requestID: 92220f15-633f-4b9c-a544-57b9b8228ec3
+        requestParameters:
+            CreateTrafficMirrorFilterRuleRequest:
+                ClientToken: b59974fd-e63c-489a-9011-4386f541e2e7
+                DestinationCidrBlock: 0.0.0.0/0
+                Protocol: 6
+                RuleAction: accept
+                RuleNumber: 100
+                SourceCidrBlock: 0.0.0.0/0
+                TrafficDirection: egress
+                TrafficMirrorFilterId: tmf-010db9a7d8056cc2d
+        responseElements:
+            CreateTrafficMirrorFilterRuleResponse:
+                clientToken: b59974fd-e63c-489a-9011-4386f541e2e7
+                requestId: 92220f15-633f-4b9c-a544-57b9b8228ec3
+                trafficMirrorFilterRule:
+                    destinationCidrBlock: 0.0.0.0/0
+                    protocol: 6
+                    ruleAction: accept
+                    ruleNumber: 100
+                    sourceCidrBlock: 0.0.0.0/0
+                    trafficDirection: egress
+                    trafficMirrorFilterId: tmf-010db9a7d8056cc2d
+                    trafficMirrorFilterRuleId: tmfr-01669e70d44b9705a
+                xmlns: http://ec2.amazonaws.com/doc/2016-11-15/
+        sessionCredentialFromConsole: true
+        sourceIPAddress: AWS Internal
+        userAgent: AWS Internal
+        userIdentity:
+            accessKeyId: ASIA57JLR4M2ZZDJUXY3
+            accountId: "123123123123"
+            arn: arn:aws:sts::123123123123:assumed-role/MakeStuffPublic
+            principalId: AROA57JLR4M2SBAPVC4BO:MakeStuffPublic
+            sessionContext:
+                attributes:
+                    creationDate: "2022-11-15T22:38:17Z"
+                    mfaAuthenticated: "true"
+                sessionIssuer:
+                    accountId: "123123123123"
+                    arn: arn:aws:iam::123123123123:role/MakeStuffPublic
+                    principalId: AROA57JLR4M2SBAPVC4BO
+                    type: Role
+                    userName: MakeStuffPublic
+                webIdFederationData: {}
+            type: AssumedRole
+      Name: Test-CreateTrafficMirrorFiltterRule
```

Comment:
there are two t's in filter as spelled here

Reply:
ack, removed and changes will be reflected in next update to pr